numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -55,6 +58,44 @@
 _py_decref.argtypes = [ctypes.py_object]
 _py_incref.argtypes = [ctypes.py_object]
 
+pynvjitlink_import_err = None
+try:
+    from pynvjitlink.api import NvJitLinker, NvJitLinkError
+except ImportError as err:
+    pynvjitlink_import_err = err
+
+
+def _readenv(name, ctor, default):
+    value = os.environ.get(name)
+    if value is None:
+        return default() if callable(default) else default
+    try:
+        return ctor(value)
+    except Exception:
+        warnings.warn(
+            f"Environment variable '{name}' is defined but its associated "
+            f"value '{value}' could not be parsed.\n"
+            "The parse failed with exception:\n"
+            f"{traceback.format_exc()}",
+            RuntimeWarning
+        )
+        return default
+
+
+config.ENABLE_PYNVJITLINK = False
+if _readenv("ENABLE_PYNVJITLINK", bool, False):
+    config.ENABLE_PYNVJITLINK = True
+
+
+_MVC_ERROR_MESSAGE_CU11 = (
+    "Minor version compatibility requires ptxcompiler and cubinlinker packages "
+    "to be available"
+)
+
+_MVC_ERROR_MESSAGE_CU12 = (
+    "Minor version compatibility requires pynvjitlink package to be available"
+)
```

Comment:
This might be confusing if the user requested pynvjitlink in order to use LTO - perhaps "Using pynvjitlink requires the pynvjitlink package to be available" would align more closely with the request that can't be satisfied?